gnd/src/main/java/com/google/android/gnd/persistence/local/room/ProjectEntity.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.persistence.local.room;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+import com.google.android.gnd.model.Project;
+import com.google.auto.value.AutoValue;
+import com.google.auto.value.AutoValue.CopyAnnotations;
+
+@AutoValue
+@Entity(tableName = "project")
+public abstract class ProjectEntity {
+
+  @CopyAnnotations
+  @NonNull
+  @PrimaryKey
+  @ColumnInfo(name = "id")
+  public abstract String getId();
+
+  @CopyAnnotations
+  @Nullable
+  @ColumnInfo(name = "title")
+  public abstract String getTitle();
+
+  @CopyAnnotations
+  @Nullable
+  @ColumnInfo(name = "description")
+  public abstract String getDescription();
+
+  @CopyAnnotations
+  @ColumnInfo(name = "is_active")
+  public abstract boolean isActive();
```

Comment:
We already store the active project in the Preference object, so it would be redundant here and would require extra logic to update the Project on activation. Or is there another reason to have it here?

Reply:
I added this property so that we can save all available projects and activate the current project. Having this property would make querying easy for last active project. But like you said, we can do that with the id saved in prefs. So, removing it now.